erela.cli.ts

```diff
@@ -197,11 +202,10 @@ const genreateApp = async () => {
     log(`Created a project ${nameWithPath}`.green.bold)
     const toCp = [
       'package.json',
-      'package-lock.json',
       '.env.example',
-      '.gitignore',
       'tsconfig.json',
       'tslint.json',
+      'README.md',
```

Comment:
Might be cool to have the project name  that got selected by the user at the top of the readme